_includes/header.html

```diff
@@ -42,7 +42,10 @@
                         </ul>
                     </li>
                     <li>
-                        <a href="/news" title="News about ev3dev">News</a>
+                        <a href="/support" title="Get Help">News</a>
```

Comment:
I'm not sure what's more worrying: the fact that I made the mistake in the first place, or the fact that I have been staring at a header with two "news" items for the past week without noticing :blush: